mozilla-release/browser/app/profile/firefox.js

```diff
@@ -1272,7 +1272,8 @@ pref("services.sync.prefs.sync.browser.safebrowsing.malware.enabled", true);
 pref("services.sync.prefs.sync.browser.search.update", true);
 pref("services.sync.prefs.sync.browser.sessionstore.restore_on_demand", true);
 pref("services.sync.prefs.sync.browser.startup.homepage", true);
-pref("services.sync.prefs.sync.browser.startup.page", true);
+pref("services.sync.prefs.sync.browser.startup.restoreTabs", true);
+pref("services.sync.prefs.sync.browser.startup.addFreshTab", true);
```

Comment:
cant we add something like:
4 - FreshTab
5 - restoreTabs and freshtab
technically restoreTabs is option 3 from http://kb.mozillazine.org/Browser.startup.page right?

if we choose to add different preferences we would simply move the complexity from the browser preferences to the extension